tmt/steps/provision/podman.py

```diff
@@ -153,7 +153,10 @@ def ansible(self, playbook):
 
     def podman(self, command, **kwargs):
         """ Run given command via podman """
-        return self.run(['podman'] + command, shell=False, **kwargs)
+        if isinstance(command, str):
+            return self.run('podman ' + command, shell=True, **kwargs)
+        else:
+            return self.run(['podman'] + command, shell=False, **kwargs)
```

Comment:
What is the motivation for this change? The `shell=False` approach should be a bit safer.

Reply:
I've let there this code, because in original source i've used it to implement behaviour for pull/push files to container and shell were easier to pass options, than try to parse tem somehow. This part can be removed.